src/index.html

```diff
@@ -5,9 +5,59 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div class="card" data-qa="card">
+
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+      >
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina
+        <br> 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__code">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
      </h2>
      
      <p class="card__code">
```

Reply:
виправив